utils/music_crawlers.py

```
@@ -376,26 +377,38 @@ async def _check_vip_status(self):
+            self._vip_checked = True
+            
+            if self._is_vip:
+                logger.info(f"[{self.platform_name}] VIP 身份探测成功 (VipType:{vip_type}, Assoc:{is_assoc_vip})")
+            elif code != 200:
+                logger.warning(f"[{self.platform_name}] 凭证失效或接口异常 (code: {code})")
+                self._cookie_invalid = True
+                self._vip_checked = False
+            else:
+                logger.info(f"[{self.platform_name}] 确认为普通账号 (无有效会员特征)")
+                logger.debug(f"[{self.platform_name}] 响应结构摘要: {list(data.keys())}")
         except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)